packages/core/src/tools/error.ts

```diff
@@ -65,3 +74,26 @@ export function toStackTraceString(stack: StackTrace) {
 export function formatErrorMessage(stack: StackTrace) {
   return `${stack.name || 'Error'}: ${stack.message!}`
 }
+
+/** Has to be called at the utmost position of the RUM call stack */
```

Comment:
what is the issue otherwise?

Reply:
`createHandlingStackTrace` go back in the stacktrace to only keep user calls. 
If we move the createHandlingStackTrace  function call, then the calculation will be wrong and we will have RUM internal calls inside the `handling stacktrace`.

Reply:
Could it be doable/interesting to be able to call `createHandlingStackTrace` from anywhere and remove RUM internal  calls ourself?

Reply:
I already tried it but I found no ways to identify if a call come from our code base or not.

- I tried looking at files name but they can change regarding how the user bundles is solution with RUM. 
- I tried looking at functions name, but they can be updated and/or have the same name twice in the stack. The dipper the handling stack is generated the more uncertainty we will have.
- Finally, the best solution I found, is to create the stack trace at the utmost point in our call stack and to remove the last call. This is the less fickle implementation which is also used in tracekit 
https://github.com/DataDog/browser-sdk/blob/c7d3145240054d9dd115c205912e84c3f59f1339/packages/core/src/domain/tracekit/computeStackTrace.ts#L573-L574

Reply:
OK, I'd be in favor of adding some of this explanation in the function comment as well